sympy/__init__.py

```diff
@@ -20,6 +27,8 @@
     pass
     # Here we can also check for specific Python 3 versions, if needed
 
+sys.modules['sympy.mpmath'] = mpmath
```

Comment:
I would not put this here. I would tell users that they can put this into their own code if needed, but sympy itself shouldn't have it.